dist/injector.js

```diff
@@ -82,8 +82,11 @@ window.FAI.bundleHost = window.FAI.bundleHost || '/';
     scanForFeature = function () {
         jQuery('[data-bundle]').each(function (idx, el) {
             var resource = jQuery(el).data().bundle,
-                host =  jQuery(el).data().host;
-            if (host) {
+                host =  jQuery(el).data().host || '',
+                source  =  jQuery(el).data().source || '';
+            if (source) {
+                NS.INJECTOR.loadFeatureForSource(resource, source);
+            } if (host) {
                 NS.INJECTOR.loadFeatureForHost(resource, host);
             } else {
                 NS.INJECTOR.loadFeature(resource);
```

Comment:
Would this logic try to load multiple resources, one from the host and one from the source?